spyder/app/tour.py

```diff
@@ -1255,6 +1255,76 @@ def any_has_focus(self):
              self.tips.hasFocus() or self.canvas.hasFocus())
         return f
 
+
+class OpenTourDialog(QDialog):
+    """Initial Widget with tour"""
+    sig_launch_tour_button_clicked = Signal()
+    # Signal triggered for the 'Dismiss' button
+    sig_dismiss_button_clicked = Signal()
+

```

Comment:
```suggestion
```

Currently, these signals aren't actually used. Unless they are needed for what @ccordoba12 suggested above, they should be removed.